draft-ietf-jose-fully-specified-algorithms.xml

```diff
@@ -223,51 +223,56 @@
 
     </section>
 
-		<section anchor="fully-specified-encryption-algs" title="Fully-specified Encryption Algorithm Identifiers">
+    <section anchor="fully-specified-encryption-algs" title="Fully-specified Encryption Algorithm Identifiers">
       <t>
-	This section describes the construction of fully-specified encryption algorithm identifiers in the context of existing JOSE and COSE encryption schemes,
-	such as JSON Web Encryption (JWE) as described in <xref target="RFC7516"/> and COSE Encrypt as described in <xref target="RFC9052"/>.
+  This section describes the construction of fully-specified encryption algorithm identifiers in the context of existing JOSE and COSE encryption schemes,
+  such as JSON Web Encryption (JWE) as described in <xref target="RFC7516"/> and COSE Encrypt as described in <xref target="RFC9052"/>.
       </t>
 
-			<t>
+      <t>
 JOSE and COSE support several modes of encryption, although the terminology may differ between JOSE and COSE,
 these modes are meant to support:
       </t>
-			<list style="symbols">
-				<t>
+      <list style="symbols">
+        <t>
 Direct Encryption - A symmetric cryptographic operation.
-				</t>
-				<t>
+        </t>
+        <t>
 Key Establishment and Direct Encryption - An assymetric cryptographic operation to derive a shared secret, key derivation and then a symmetric cryptographic operation.
-				</t>
-				<t>
+        </t>
+        <t>
 Two Layer Encryption - A content encryption key is protected (many possible ways), content encryption and decryption is performed using the protected content encryption key.
-				</t>
-			</list>
-			<t>
+        </t>
+      </list>
+      <t>
 Mode complexity creates the following risks:
-			</t>
+      </t>
 
-			<list style="symbols">
-				<t>
+      <list style="symbols">
+        <t>
 The combination of chosen algorithms might not be implemented by the receiver.
-				</t>
-				<t>
+        </t>
+        <t>
 The combinations of chosen algorithms migth not be aligned in terms of bit strength.
-				</t>
-				<t>
+        </t>
+        <t>
 Under specified or implicit parameters could lead to exploitable faults in implementations, for example cross curve ECDH between P-256 and P-384 or X25519.
-				</t>
-				<t>
+        </t>
+        <t>
 Alternative algorithms at a component layer, such as symmetric key encryption, might provide different security properties, for example AES-CBC vs AES-GCM.
-				</t>
-			</list>
+        </t>
+      </list>
 
-			<t>
+      <t>
+While this specification provides a definition of what fully specified encryption algorithm identifiers are for both JOSE and COSE, including examples,
+it does not deprecate any polymorphic algorithms or register any new fully specified algorithms.
```

Comment:
I think we should register a small number of fully-specified encryption algorithms, but I'm OK leaving this text as-is and changing it when we add those registrations.